global_settings/migrations/0007_footer.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.6 on 2017-08-22 14:48
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('wagtailcore', '0032_add_bulk_delete_page_permission'),
+        ('global_settings', '0006_auto_20161213_1054'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Footer',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('supporters', models.CharField(max_length=255)),
```

Comment:
I take `supporters` to be the paragraph that starts with "Supported by Rice University...". 255 chars won't quite hold what we have there now.

Reply:
This is actually an old migration, which is updated later in this PR. If you look at the models.py file, you'll see all these have been changed to TextFields, so they should support enough characters.